stablehlo/reference/Interpreter.cpp

```diff
@@ -0,0 +1,191 @@
+/* Copyright 2022 The StableHLO Authors.
```

Comment:
`Interpreter.cpp` was deleted in the previous PR and merged into Ops.cpp, so you can delete this file.

Reply:
Weird it's back. Deleted :-)